### ISSUE TITLE
ipfshttpclient doesn't support IPFS 0.6.0

### ISSUE THREAD

**Issue by Vourhey** [ipfs-shipyard/py-ipfs-http-client#221]
Recently [IPFS 0.6.0](https://github.com/ipfs/go-ipfs/releases/tag/v0.6.0) was released but ipfshttpclient doesn't support it

```
>>> import ipfshttpclient
>>> client = ipfshttpclient.connect()
Traceback (most recent call last):
  File "/home/vadim/.local/lib/python3.8/site-packages/ipfshttpclient/http.py", line 266, in _do_raise_for_status
    response.raise_for_status()
  File "/usr/lib/python3.8/site-packages/requests/models.py", line 941, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 405 Client Error: Method Not Allowed for url: http://localhost:5001/api/v0/version?stream-channels=true

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/vadim/.local/lib/python3.8/site-packages/ipfshttpclient/client/__init__.py", line 101, in connect
    version_str = client.version()["Version"]
  File "/home/vadim/.local/lib/python3.8/site-packages/ipfshttpclient/client/base.py", line 15, in wrapper
    result = func(*args, **kwargs)
  File "/home/vadim/.local/lib/python3.8/site-packages/ipfshttpclient/client/miscellaneous.py", line 204, in version
    return self._client.request('/version', decoder='json', **kwargs)
  File "/home/vadim/.local/lib/python3.8/site-packages/ipfshttpclient/http.py", line 51, in wrapper
    return func(self, *args, **merged)
  File "/home/vadim/.local/lib/python3.8/site-packages/ipfshttpclient/http.py", line 385, in request
    res = self._request(method, url, params, stream, files, headers,
  File "/home/vadim/.local/lib/python3.8/site-packages/ipfshttpclient/http.py", line 307, in _request
    self._do_raise_for_status(res)
  File "/home/vadim/.local/lib/python3.8/site-packages/ipfshttpclient/http.py", line 286, in _do_raise_for_status
    six.raise_from(exceptions.StatusError(error), error)
  File "<string>", line 3, in raise_from
ipfshttpclient.exceptions.StatusError: HTTPError: 405 Client Error: Method Not Allowed for url: http://localhost:5001/api/v0/version?stream-channels=true
```

**Comment by Vourhey** [comment]
Also I got the following exception

```
ipfshttpclient.exceptions.VersionMismatch: Unsupported daemon version '0.6.0' (not in range: 0.4.21 ≤ … < 0.6.0)
```

**Comment by ibnesayeed** [comment]
We too are getting the VersionMismatch error in https://github.com/oduwsdl/ipwb/pull/670.

**Comment by ntninja** [comment]
Fixed in 0.6.0
